doc/configuration.rst

```diff
@@ -605,13 +605,13 @@ NetworkRKUSBLoader
 ~~~~~~~~~~~~~~~~~~~
 A NetworkRKUSBLoader describes an `RKUSBLoader`_ available on a remote computer.
 
-AndroidFastboot
-~~~~~~~~~~~~~~~
-An AndroidFastboot resource describes a USB device in the fastboot state.
+AndroidUsbFastboot
```

Comment:
This is a breaking change, we want to retain backwards compatibility with a DeprecationWarning, than we can remove `AndroidFastboot` in a later release.

Reply:
Ok. I changed the PR and add a `DeprecationWarning` like in [`usbstoragedriver.py`](https://github.com/labgrid-project/labgrid/blob/dffc61d31db002d674ce2a31cff9795760d65682/labgrid/driver/usbstoragedriver.py#L161)